src/wovnio/wovnphp/RequestOptions.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Wovnio\Wovnphp;
+
+class RequestOptions
+{
+    /*
+     * disableMode:
+     *      - do nothing to the request
+     */
+    private $disableMode;
+    /*
+     * cacheDisableMode:
+     *      - bypass cache for request to translation API
+     * Only available if debugMode is also turned on server side.
+     */
+    private $cacheDisableMode;
+    /*
+     * debugMode:
+     *      - activate extra debugging information.
+     *      - send "debugMode=true" to translation API
+     *      - bypass cache for request to translation API
+     * Only available if debugMode is also turned on server side.
+     */
+    private $debugMode;
+
+
+    public function __construct($queryString, $debugModeEnable)
+    {
+        $this->disableMode = false;
+        $this->cacheDisableMode = false;
+        $this->debugMode = false;
+
+        if ($queryString !== null) {
+            $this->disableMode = strpos($queryString, 'wovnDisable') !== false;
```

Comment:
note that this is a bit naive. We would like to allow `wovnDisable=false/true` ideally. Which is why making `$headers->query` public (+ https://www.php.net/manual/en/function.parse-str.php perhaps to turn it into an array) might be a better approach.

Reply:
I can do that, though I'm not sure when we'd ever want to explicitly set wovnDisable to false.

Reply:
It's handy when you're debugging to switch between on/off quickly

Reply:
Didn't realize wovnjava also behaves like this though.

Reply:
Yeah that's why I did it that way 🙃